zfec/test/test_zfec.py

```diff
@@ -286,40 +293,40 @@ def _help_test_filefec(self, teststr, k, m, numshs=None):
             tempdir.shutdown()
 
     def test_filefec_all_shares(self):
-        return self._help_test_filefec("Yellow Whirled!", 3, 8)
+        return self._help_test_filefec(b"Yellow Whirled!", 3, 8)
```

Comment:
maybe prettier than using b"..." at every place here would be to change self._help_test_filefec() to convert to bytes_type (which needs to be set up in a similar way as text_type, str for py2, bytes for py3).